src/main/java/version1/gameUtil/settings/UIConfigurations.java

```diff
@@ -159,6 +159,7 @@ public static void configure(ModeScreen modeScreen){
                 .createHardButton(hardButtonText)
                 .createGoToMenuButton(goToMenuText)
                 .createGoToMediumMazeButton(mediumButtonText)
+                .createGoToHardMazeButton(hardButtonText)
```

Comment:
Hi @cammurriel I see you have created a `createGoToHardMazeButton()` on line 162, and it seems to me like it is doing the same thing that the createHardButton() is doing on line 159.

I think it was my bad for not telling you that the methods were already created and that you just needed to fill them in.